src/chesster.js

```diff
@@ -850,8 +850,8 @@ function schedulingReplyAmbiguous(bot, message){
 
 // Game has been scheduled.
 function schedulingReplyScheduled(bot, message, results, white, black) {
-    var whiteDate = results.date.clone().utcOffset(white.tz_offset/60);
-    var blackDate = results.date.clone().utcOffset(black.tz_offset/60);
+    var whiteDate = results.date.clone().tz(white.tz);
+    var blackDate = results.date.clone().tz(black.tz);
```

Comment:
I've removed the message. This is as accurate as we can make it. The only reason we will be wrong is if the person gets their slack timezone wrong. I'm ok with that situation.